gammapy/data/gti.py

```diff
@@ -306,36 +306,48 @@ def from_time_intervals(cls, time_intervals, reference_time=None):
 
         return cls.create(start, stop, reference_time)
 
-    def select_time(self, time_interval):
+    def select_time(self, time_intervals):
```

Comment:
technically you are implementing an `intersection` method, right?

Reply:
yes, indeed... the intersection between the observattory GTIs and the user's time intervals